packages/mesos/buildinfo.json

```diff
@@ -3,7 +3,7 @@
   "single_source" : {
     "kind": "git",
     "git": "https://github.com/mesosphere/mesos",
-    "ref": "6275099d4d5e0eadc5d39cf7a28d6d7930788680",
+    "ref": "80928eaf1d66e2d06d4f301ac197c12d08eaa533",
```

Comment:
@greggomann not sure what the right approach is here, but do we want to add another cherry-pick or does it make sense to just merge the change into the `Mesos UI` cherry-pick (`202dc73803a3e681e31dd30ca68a44fdae2ba902`) ? I want to limit the number of cherry-picks we have to specific components. Will reduce the error when we apply these to new bumps?